card/lib/cardio.rb

```diff
@@ -50,26 +48,19 @@ def set_config config
   
     def set_paths paths
       @@paths = paths
-      paths.add 'tmp/lib'
-      paths.add 'tmp/set'
-      paths.add 'tmp/set_pattern'
-      paths.add 'db/migrate_deck_cards', :with=>'db/migrate_cards'
+      add_gem_path 'tmp/set', :root => root
+      add_gem_path 'tmp/set_pattern', :root => root
```

Comment:
That seems about right.  I didn't think I added the handling of the root option.

You are saying rename add_gem_path because it is more general, not having two methods, right?